base/show.jl

```diff
@@ -926,7 +926,7 @@ function show_block(io::IO, head, args::Vector, body, indent::Int)
         end
     end
 
-    ind = head === :module || head === :baremodule ? indent : indent + indent_width
+    ind = head === :module || head === Symbol("bare module") ? indent : indent + indent_width
```

Comment:
I don't think this expression head ever occurs (true of `:baremodule` as well, I believe).

Reply:
I think it is generated from this line: https://github.com/JuliaLang/julia/pull/26494/files/bb6996198aa841e60b5238b2269d979bb2c0d037#diff-b4267643a238b5a375b3e055817ee6caR1322.